docs/api/field_type_reference.md

```diff
@@ -1937,12 +1937,6 @@ $validators = [
 
 ## RichText Field Type
 
-!!! caution "Deprecated"
-
-    Make sure to enable new version of the RichText Field Type provided via [eZ Platform RichTextBundle](https://github.com/ezsystems/ezplatform-richtext).
```

Comment:
Maybe we can rephrase this to state that RichText Field Type is provided `via [eZ Platform RichTextBundle](https://github.com/ezsystems/ezplatform-richtext)`?